### ISSUE TITLE
Add option to create a new page when arriving at a 404

### ISSUE THREAD

**Issue by nikolaiwarner** [kodedninja/orkl#12]
On the 404 page, it would nice if there was a button to make a new page based on that path... much like the feature often found in wikis.

**Comment by kodedninja** [comment]
Do you mean if the owner of the site arrives to a 404 page? I like the idea, though!

**Comment by nikolaiwarner** [comment]
Yes! I found myself making pages rapidly and wanting to just make links before I made the page.

**Comment by kodedninja** [comment]
Gotcha! On todo. Hopefully holidays will leave some time to be productive haha
